CI_README.md

```diff
@@ -0,0 +1,51 @@
+GitHub CI tools for MagicModules and Google Providers
+===
+
+These tools manage the downstream repositories of [magic-modules](https://github.com/GoogleCloudPlatform/magic-modules), and are collectively referred to as "The Magician".
+
+# CI For Downstream/Magic Modules Developers
+If you're interested in developing the repositories that Magic Modules manages, here are the things you'll want to know.
+
+## What The Magician Does
+The Magician takes the PR you write against MagicModules and creates the downstream (generated) commits that are a result of your changes.  It posts the diffs created by your changes as a comment on your PR, to aid in review.  When your PR is merged, it updates the downstream repositories with your changes.
+
+## Your Workflow
+
+You'll write some code and open a PR.  The Magician will run the generator and the downstream tests.  If either the generator or the tests fail, you'll see "check failed" status on your PR, which will link to the step that failed.  Once all the generation steps succeed, the Magician will comment on your PR with links to review the generated diffs.  Your reviewer will review those diffs (as well as your code).  Once your PR is approved, simply merge it in - the Magician will ensure that the downstream repositories are updated within about 20 minutes.
```

Comment:
> The Magician will run the generator and the downstream tests

Where is this defined in the workflow configs? I'm not seeing them.

Reply:
Ah, I removed the tests to speed it up.  /facepalm.  I'll get it back.